standards/node.md

```diff
@@ -0,0 +1,29 @@
+## Draft Node.js Standards
+
+### General
+* Node.js code is JS code and should follow the [JS standards](javascript.md)
+* Session state should not be stored on the node app server. Don't tie a session to a particular node server instance. Use a distributed cache or document storage database and not something like express-session. 
+* Avoid blocking the [main event loop and the worker pool](https://nodejs.org/en/docs/guides/dont-block-the-event-loop/). In short "you shouldn't do too much work for any client in any single callback or task." and consider passing CPU intensive tasks off to another service.
+* Prefer await over callbacks and avoid nested callbacks. This is easily done in [Node 8 and above](https://nodejs.org/api/util.html#util_util_promisify_original).
+
+### Versions
+
+* Be aware of the [Node.js support timeline](https://nodejs.org/en/about/releases).
+* Keep on Active LTS versions.
+* Don't drop behind Maintenance LTS versions. Projects older than this will be considered unmaintainable and
+  unsupportable until brought up to an appropriate version.
+* Don't progress beyond Active LTS versions.
+
+### Package Management
+* Use NPM.
+* Use a package.json and package-lock.json for repeatable builds.
+* Use either a vulnerability scanner such as GreenKeeper or an update tool such as dependabot.
```

Comment:
I'm being pedantic now 😁 

Both [Greenkeeper](https://greenkeeper.io/) and [Dependabot](https://dependabot.com/) are automated dependency update tools. However, Greenkeeper is NPM only, whereas Dependabot covers multiple languages including Ruby Gems, Java Maven, and .NET. Plus its been bought by GitHub, so we expect sometime soon its functionality will just be rolled into GitHub.

That's why in Bristol we have tried to make using Dependabot a standard across all projects and moved away from Greenkeeper.

For vulnerability scanning, we opted for [Snyk](https://snyk.io/) again because it's multi-language and supports our chosen technologies, plus Ruby and Java.

Oh, and they are all free for open source 🎉

Reply:
Corrected the wording - hope this is better!

Reply:
That works, but as this is the 'standard' fancy being nudged to just pick one to be our _standard_ dependency checker 😉 !?